prow/resultstore/files.go

```diff
@@ -34,13 +34,23 @@ type fileFinder interface {
 	Attributes(ctx context.Context, name string) (pio.Attributes, error)
 }
 
+// MustFile describes a file that should exist in ArtifactsOpts.Dir.
+// If the file is not present, these values will be used instead.
+type MustFile struct {
```

Comment:
Nit: Is "Must" a keyword already used in ResultStore? If not, I think there might be a better name here because "Must" by itself doesn't convey too much meaning. How about  "DefaultFile", "PromisedFile", "ExpectedFile", or "EagerFile"?

Reply:
I haven't noticed "Must" in any identifiers in the resultstore protos, but I believe the naming could be better. I'll go with DefaultFile which seems clearest. Thanks!